python/taichi/ui/scene.py

```diff
@@ -106,7 +116,39 @@ def lines(self,
                 If `per_vertex_color` is provided, this is ignored.
             per_vertex_color (Tuple[float]): a taichi 3D vector field, where each
                 element indicate the RGB color of the line.
+            vertex_offset: int type(otherwise float type will be floored to int),
+                if 'indices' is provided, this means the value added to the vertex
+                index before indexing into the vertex buffer, else this means the
+                index of the first vertex to draw.
+            vertex_count: int type(ohterwise float type will be floored to int),
+                only avaliable when `indices` is not provided, which is the number
+                of vertices to draw. There are 2 cases that we will change your
+                vertex_count. [1] If the `vertex_count` is a odd number, then we
+                will change it to `vertex_count` - 1. [2] If `vertex_offset` plus
+                `vertex_count` greater than vertices.shape[0], then we will reduce
+                `vertex_count` to no more than vertices.shape[0].
```

Comment:
vertex_count: int type (otherwise float type will be floored to int),
                only available when `indices` is not provided, which is the number
                of vertices to draw. There are 2 cases where we will change your
                vertex_count. [1] If the `vertex_count` is an odd number, then we
                will change it to `vertex_count` - 1. [2] If `vertex_offset` plus
                `vertex_count` is greater than vertices.shape[0], then we will reduce
                `vertex_count` to no more than vertices.shape[0].

Reply:
Thanks for you suggestions!